src/actions/github.ts

```diff
@@ -0,0 +1,63 @@
+export const parseOAuthCode = () => {
+    const windowUrl = window.location.search;
+    const params = new URLSearchParams(windowUrl);
+
+    const code = params.get("code");
+
+    return code
+}
+
+const callGatekeeper = (code: string) => {

```

Comment:
i think it's a good standard IMO, is to pass params as an object ({ code }: Props)
this way other devs can put in params in any order

Reply:
Done!